extensions/amp-animation/amp-animation.md

```diff
@@ -82,7 +82,7 @@ and is comprised of:
  - Timing properties
  - Keyframes
 
-```
+```text
```

Comment:
I'll investigate further and put together something so we all know what formats are acceptable.  Also, Grow SDK is fixing backticks with no code specified - default to text.  Just not in the release we use.  I'll test it out when the release is available.